engine/common/hpak.c

```diff
@@ -17,8 +17,8 @@ GNU General Public License for more details.
 #include "hpak.h"
 
 #define HPAK_MAX_ENTRIES	0x8000
-#define HPAK_MIN_SIZE	(1 * 1024)
-#define HPAK_MAX_SIZE	(128 * 1024)
+#define HPAK_ENTRY_MIN_SIZE	(512)
```

Comment:
Do we want to have minimal entry size here?

Reply:
You mean we just can set minimal entry size 0 and just remove this macro?